css/style.css

```diff
@@ -138,6 +138,13 @@
 	font-weight: bold;
 }
 
+.activity-section a:hover,
+.activity-section a:focus,
```

Comment:
text looks really awkward now since the grey is very slightly off. Could we use like `.7` which actually matches the normal font color? For the previews the `.5` looks okay

Reply:
@nickvergessen what do you mean by »slightly off« and not matching the normal font color? .7 is too little difference between the normal state, and keep in mind this is only on hover and focus. :)

Reply:
> ![bildschirmfoto vom 2017-10-13 12-07-14](https://user-images.githubusercontent.com/213943/31541500-2444842e-b00f-11e7-9c54-197dfaec067a.png)

Yeah the difference between a hovered link and the rest of the text looks awkward I think

Reply:
I adjusted the hover opacity to .7 while keeping the rest at .5 :)